pages/linux/pkgfile.md

```diff
@@ -16,18 +16,14 @@
 
 `pkgfile --list {{package_name}}`
 
-- List only files in the `bin` directory provided by a package:
+- List only files provided by a package located within the `bin` or `sbin` directory:
 
 `pkgfile --list --binaries {{package_name}}`
 
 - Search for a package that owns a specific file using case insensitive matching:
 
 `pkgfile --ignorecase {{filename}}`
 
-- Search for a package that owns a specific file in the `bin` directory:
-
-`pkgfile --binary {{filename}}`
-
```

Comment:
@peterbabic Maybe this was just a typo in my suggestion but `--binaries` works as the example description says:
```
$ pkgfile --binaries node
community/nodejs
community/nodejs-lts-dubnium
community/nodejs-lts-erbium
community/nodejs-lts-fermium
$ pkgfile --binaries bat
community/bat
```

Reply:
@navarroaxel I can confirm. I could have tested that, my bad.

Apart from fixing this typo, are there any other changes worth keeping, like the link to Arch man or `sbin` mention (`sbin` mention would apply to this description as well). Also note that on my system `/usr/sbin` is symlinked to `/usr/bin` so no much value in mentioning `sbin` separately, if other Arch based systems have it the same way. 

You guys decide, I am not entirely sure how to revert effectively yet, although I could figure it out.

Reply:
I have the symlink too. I think we can keep that reword.
Just restore the deleted example.